runtime/doc/options.txt

```diff
@@ -2162,7 +2173,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 	   fuzzy    Enable |fuzzy-matching| for completion candidates. This
 		    allows for more flexible and intuitive matching, where
 		    characters can be skipped and matches can be found even
-		    if the exact sequence is not typed.
+		    if the exact sequence is not typed. this option works more
+		    like fuzzy filter, for some default |ins-completion| modes
+		    like keyword, filenames wholelines see
+		    |completefuzzycollect|
```

Comment:
The newly added description is not clear enough, recommend more details, such as `Note: This option does not affect the collection of candidate list, it only controls how completion candidates are reduced from the list of alternatives. If you want to use |fuzzy-matching| to gather more alternatives for your candidate list, see |completefuzzycollect|.`